src/app/api/github/webhook/route.ts

```
@@ -0,0 +1,80 @@
+            throw new Error('GitHub token not configured');
+        }
+
+        const commits = await fetchGitHubCommits(repoUrl, githubToken);
+        const analysis = analyzeCommitHistory(commits);
+
+        // 3. Update submission status based on analysis
+        const newStatus = analysis.isSuspicious ? 'pending_audit' : 'attended';
+
+        const { error: updateError } = await supabase
+            .from('user_series_progress')
+            .update({
```

ESLint

[error] Delete `······`

(prettier/prettier)